ktor-client/ktor-client-okhttp/jvm/test/io/ktor/client/engine/okhttp/RequestTests.kt

```diff
@@ -4,12 +4,32 @@
 
 package io.ktor.client.engine.okhttp
 
+import io.ktor.application.*
+import io.ktor.client.*
 import io.ktor.client.request.*
 import io.ktor.client.tests.utils.*
+import io.ktor.http.*
+import io.ktor.network.sockets.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import io.ktor.server.cio.*
+import io.ktor.server.engine.*
+import kotlinx.coroutines.*
 import okhttp3.*
+import java.time.*
 import kotlin.test.*
 
-class RequestTests {
+class RequestTests : TestWithKtor() {
+
+    override val server = embeddedServer(CIO, serverPort) {
```

Comment:
Please rewrite this suite using `ClientLoader` and move the test in common

Reply:
This issue is related to OkHttp only and is not reproduced with common configuration